### DISCUSSION TITLE
iOS automation shortcuts

### DISCUSSION THREAD

**Discussion by jeremygaither** [simple-login/app#495] [Feature Requests]
When using a different email privacy service and their app, I was able to use iOS Shortcuts to automate the process of creating an email alias to one step. SimpleLogin ios already has some smarts in this area when sending the page to the app, but I would prefer a little more control.

I've published my shortcut here, which may help you elicit some use cases and stories: https://routinehub.co/shortcut/9051/

**Comment by frederikkrohn** [comment]
Yes @nguyenkims please! 🙌
